frontend/src/styles/_uswds-theme.scss

```diff
@@ -39,6 +39,14 @@ in the form $setting: value,
   $theme-column-gap-desktop: 6,
   $theme-column-gap-md: 4,
   $theme-column-gap-lg: 6,
+  $position-settings: (
+    output: true,
+    responsive: true,
+    active: false,
+    focus: false,
+    hover: false,
+    visited: false
+  ),
```

Comment:
This is required to add positioning only on larger screens. I'm only changing the `responsive` values, but it only seems to work if you include all position settings. I think this happened before with breakpoints. I'm confused as to when you can override an individual setting and when you have to include all. Maybe we can get clarity on this?

Reply:
I'm not sure either, that's so interesting. I thought you'd be able to do that.